app/javascript/components/widgets/weather/minutely-weather.jsx

```diff
@@ -1,19 +1,34 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import gql from 'graphql-tag';
+import styled from 'styled-components';
 import withFragment from '../../hocs/with-fragment';
+import SkyIcon from './sky-icons';
 
 const getMinutelyWeather = gql`
   fragment MinutelyWeather on Weather {
     minutely {
       summary
+      icon
     }
   }
 `;
 
-const MinutelyWeather = ({ weather }) => {
+const IconWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+const MinutelyWeather = ({ weather, width, height }) => {
   const { summary } = weather.minutely;
-  return <span>{summary}</span>;
+  const { icon } = weather.minutely;
+  return (
+    <IconWrapper>
+      <SkyIcon icon={icon} width={width} height={height} />
```

Comment:
I feel like there is an opportunity here to specify intent rather than height/width e.g. `<MinutelyWeather useLargeIcon={true|false} />`

```
{ useLargeIcon ? <SmallSkyIcon icon={icon} : <LargeSkyIcon icon={icon} }
```